std_msgs/Header header

geometry_msgs/Point norm_pos
float64 confidence
